keras_cv/models/mobilenet_v3.py

```diff
@@ -112,48 +112,44 @@ def depth(x, divisor=8, min_value=None):
     return new_x
 
 
-def HardSigmoid(name=None):
+def HardSigmoid(x, name=None):
```

Comment:
let's rename these activation functions (and the InvertedResBlock) to use the `apply_xyz` convention

(e.g. `apply_hard_sigmoid`, `apply_inverted_res_block`)